melodies_monet/driver.py

```diff
@@ -409,6 +419,7 @@ def __init__(self):
         """dict : Paired data, set by :meth:`pair_data`."""
         self.start_time = None
         self.end_time = None
+        self.time_intervals = None
```

Comment:
- self.time_intervals should be returned by __repr__(self) in analysis (around line 436).

Reply:
Agree with @colin-harkins, we do not want to truncate the complete analysis period.  In that example the fractional day should be included as the last time interval ... will address this ...